pkg/httpclient/externalRequestClient.go

```diff
@@ -38,3 +40,14 @@ func (hcb *ClientBuilder) Build() mobile.ExternalHTTPRequester {
 	client.Timeout = time.Second * time.Duration(hcb.timeout)
 	return client
 }
+
+func CloseResponseBody(res *http.Response, logger *logrus.Logger) {
```

Comment:
is this passing the new go-fmt test? It's missing a comment anyway

Reply:
go fmt doesn't care about comments that is golint

Reply:
Meant to remove that function actually